app/templates/components/post-header.hbs

```diff
@@ -1,2 +1,2 @@
 <div class="post-icon"></div>
-{{post-title post=post}}
+{{post-title post=post saveTitle=saveTitle}}
```

Comment:
If you already know the ins-and-outs of how ember works, it's not a problem, but if you're new and trying to figure out how things work, it's a nightmare.